### ISSUE TITLE
Bias is ignored while loading models

### ISSUE THREAD

**Issue by whwang1996** [noon99jaki/pra#7]
During loading models, bias is ignored and set to zero due to the third parameter here is true. Is that normal?
https://github.com/noon99jaki/pra/blob/cf02b24a6a50347cf59bbb307941c429a946f8ef/edu/cmu/lti/algorithm/optimization/AModel.java#L43

Thanks!

**Comment by noon99jaki** [comment]
Yeah, it does seems a bit counter intuitive, but since it is a ranking model the bias term should not matter. Might be an issue if we try to do classification with it.

**Comment by noon99jaki** [comment]
Yeah, it does seems a bit counter intuitive, but since it is a ranking
model the bias term should not matter. Might be an issue if we try to do
classification with it.
Ni

**Comment by whwang1996** [comment]
I see. So if I want to get the confidence that a triple exists, should I add the bias term to the score? For example, the predictions score of (h,r,t) is 3, and the bias term is -1. So the confidence of (h,r,t) exists is sigmoid(3-1)=0.881. Is that correct?

**Comment by noon99jaki** [comment]
yeah that work. However, the estimated probability is based on the distribution of the training set which might differ than your test set. So you might anyway need to calibrate your probabilities like in
https://dl.acm.org/doi/abs/10.1145/2623330.2623623
